File: src/main/java/com/key/oa/repository/PageResRepository.java
package com.key.oa.repository;

import com.key.oa.entity.PageRes;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
import org.springframework.stereotype.Repository;

/**
 * @author 孙强
 */
@Repository
public interface PageResRepository extends JpaRepository<PageRes, Long>, JpaSpecificationExecutor<PageRes> {
}

File: src/main/java/com/key/oa/repository/PageResRepository.kt
package com.key.oa.repository

import com.key.oa.entity.PageRes
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.jpa.repository.JpaSpecificationExecutor
import org.springframework.stereotype.Repository

@Repository
interface PageResRepository : JpaRepository<PageRes, Long>, JpaSpecificationExecutor<PageRes>